benchmarks/mongo-es-speed/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "mongo-es-speed",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "babel-node ./index.js",
+    "test": "exit 0"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "babel-cli": "^6.24.1",
+    "mongodb": "^2.2.26",
+    "nyan-progress": "^1.1.0",
+    "pify": "^2.3.0",
+    "pug": "^2.0.0-beta.12",
+    "resolve-es": "*",
+    "resolve-es-mongo": "*"
+  },
+  "devDependencies": {
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
The following packages are declared in the top level *package.json* and not required here.